scripts/fun_fact.coffee

```diff
@@ -0,0 +1,22 @@
+# Description:
+#   Random fun facts scraper. From http://randomfunfacts.com
+#
+# Dependencies:
+#   "cheerio": "~0.12.0"
+#
+# Commands:
+#   hubot fun fact me
+#
+# Author:
+#   monfresh
+
+cheerio = require 'cheerio'
+
+module.exports = (robot) ->
+  robot.respond /fun fact me/i, (msg) ->
```

Comment:
Minor: like a lot of the [other hubot scripts](https://github.com/hubot-scripts/hubot-google-images/blob/6c37b88ee045251b2c98a40ac41d5f5d8724fab0/src/google-images.coffee#L11), I'd make the "me" optional.